packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -269,39 +266,35 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
             collateralAvailable,
           );
 
-          const { curAuctionPrice, currencySeat, remainingProceedsGoal } =
+          const { curAuctionPrice, bidSeat, remainingProceedsGoal } =
             this.state;
           curAuctionPrice !== null ||
             Fail`auctionPrice must be set before each round`;
           assert(curAuctionPrice);
 
-          const currencyNeeded = ceilMultiplyBy(
+          const bidNeeded = ceilMultiplyBy(
             initialCollateralTarget,
             curAuctionPrice,
           );
-          if (AmountMath.isEmpty(currencyNeeded)) {
+          if (AmountMath.isEmpty(bidNeeded)) {
             seat.fail(Error('price fell to zero'));
             return makeEmpty(collateralBrand);
           }
 
-          const initialCurrencyTarget = AmountMath.min(
-            currencyNeeded,
-            currencyAlloc,
-          );
-          const currencyLimit = remainingProceedsGoal
-            ? AmountMath.min(remainingProceedsGoal, initialCurrencyTarget)
-            : initialCurrencyTarget;
+          const initialBidTarget = AmountMath.min(bidNeeded, bidAlloc);
```

Comment:
This is where using *Bid* as the token name gets confusing. This is not a target of the bidding, it's a target amount to spend.

Can I take over the PR and try to rename these variables?

Reply:
please do. I'll remove the automerge tag